<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextWindow</class>
 <widget class="QDialog" name="TextWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="commands">
     <item>
      <widget class="QLabel" name="size_label">
       <property name="text">
        <string>Font size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="font_size_box">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>6.000000000000000</double>
       </property>
       <property name="maximum">
        <double>64.000000000000000</double>
       </property>
       <property name="value">
        <double>12.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_fixed_width">
       <property name="text">
        <string>Fixed width</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_word_wrap">
       <property name="text">
        <string>Word wrap</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save_text_button">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Save text</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copy_button">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Copy</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cut_button">
       <property name="text">
        <string>Cut</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="paste_button">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Paste</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_up">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="text"/>
   </item>
   <item>
    <widget class="Line" name="line_down">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="text_legend">
     <item>
      <widget class="QLabel" name="cursor_position_label">
       <property name="text">
        <string>Cursor position</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="text_size_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>28</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <italic>false</italic>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="lineWidth">
        <number>6</number>
       </property>
       <property name="text">
        <string>Text size</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
